src/endpoints/CoverageIntegrations/types.ts

```diff
@@ -19,6 +19,7 @@ export interface CreateRequest {
 }
 
 export interface UpdateRequest {
+    newsroom?: Newsroom['id'] | Newsroom['uuid'];
```

Comment:
Please let me know if this is correct. Since this is optional I guess we can update it in update request?